api/swagger.yml

```diff
@@ -1311,6 +1311,9 @@ components:
           $ref: "#/components/schemas/StorageConfig"
         storage_config_list:
           $ref: "#/components/schemas/StorageConfigList"
+        customization:
```

Comment:
I don't believe this is the right approach.
customization under storage config is too obscure.
We should probably have a specific `ui` field with explicit `custom_viewers` under it.
Note that this was not detailed in PRD or design document - so I'm pushing back on this.

Reply:
Please see my comment in the other PR 🙏